daras_ai_v2/breadcrumbs.py

```
@@ -2,6 +2,7 @@
 
 import gooey_gui as gui
 from bots.models import SavedRun, PublishedRun
+from daras_ai.image_input import truncate_text_words
```

Ruff (0.11.9)

`daras_ai.image_input.truncate_text_words` imported but unused

Remove unused import: `daras_ai.image_input.truncate_text_words`

(F401)